src/Boleteria.java

```diff
@@ -6,6 +6,7 @@
     Map<Integer, Cliente> registroClientes;
     Map<Sala,String[]> peliculasDia;
     int precioPelicula;
+    int precio2D = 40,precio3D = 50;
```

Comment:
```suggestion
    int precio2D = 40
    int precio3D = 50;
```
Para que se vea esteticamente mejor.